test/e2e/framework/pod/delete.go

```diff
@@ -56,6 +57,20 @@ func DeletePodWithWait(ctx context.Context, c clientset.Interface, pod *v1.Pod)
 	return DeletePodWithWaitByName(ctx, c, pod.GetName(), pod.GetNamespace())
 }
 
+// DeletePodWithWaitByBatch deletes the passed-in pods, waits for the pod to be terminated, return deletion error if exists
+func DeletePodWithWaitByBatch(ctx context.Context, c clientset.Interface, pods []*v1.Pod) {
```

Comment:
nit: should this be named like this:

```suggestion
func DeletePodsWithWaitByBatch(ctx context.Context, c clientset.Interface, pods []*v1.Pod) {
```

Reply:
done